_layouts/default.html

```diff
@@ -3,36 +3,49 @@
 
   {% include head.html %}
 
-  <body>
+  <body class="no-banner">
 
-    {% include header.html %}
+    <header>
+      <div class="container">
+        {% include regions/menu.html %}
```

Comment:
I think we can wrap this with the `row` class as the main wrappers of the menu.html output.

Reply:
Copy / pasted from the original file. I will fix it, thanks.